README.md

```diff
@@ -1 +1,11 @@
-# browserstack-percy-status-sync
+# BrowserStack - Percy Status Sync
+* Mark the test status on BrowserStack based on the Percy build state.
+
+## Pre-requisites
+````
+export BROWSERSTACK_USERNAME=<YOUR_BROWSERSTACK_USERNAME> 
+export BROWSERSTACK_ACCESS_KEY=<YOUR_BROWSERSTACK_ACCESS_KEY>
+export PERCY_TOKEN=<PERCY_FULL_ACCESS_TOKEN>
+````
+## Steps to run
```

Comment:
npm install needs to be executed to install package.json dependencies

Reply:
done